### PR TITLE
feat(examples): fresh

### PR DESCRIPTION
**Pull request by notrab**
# Description

Please include a summary of the change and which issue is fixed.
Please also include relevant motivation and context.

# Type of change

- [ ] 💔 Breaking
- [ ] 🚀 Feature
- [ ] 🐛 Fix
- [ ] 🛠️ Tooling
- [ ] 🔨 Refactoring
- [ ] 🧪 Test
- [ ] 📦 Dependency
- [ ] 📖 Requires documentation update

### CODE REVIEW

**Comment by vitalikda** [comment]
`.gitingore` is missing

File: examples/fresh/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+mutation {
+  postCreate(input: { title: "Hello Fresh", slug: "hello-fresh" }) {
+    post {
+      id
+    }
+  }
+}
+```
+
+6. In another terminal, run `deno task start` and visit [`http://localhost:8000`](http://localhost:8000)
```
Comment (vitalikda) [inline examples/fresh/README.md:28]:
```suggestion
6. In another terminal, run `npx deno task start` and visit [`http://localhost:8000`](http://localhost:8000)
```

**Review by vitalikda — APPROVED** [review]
Looks great! Thank you 👏

**Comment by notrab** [comment]
> `.gitingore` is missing

@vitalikda is there anything specific you'd like me to git ignore?

**Pull request merged**